configs/datasets/test_data_sqlite.yml

```diff
@@ -1,26 +1,34 @@
-path: $GRAPHNET/data/tests/sqlite/oscNext_genie_level7_v02/oscNext_genie_level7_v02_first_5_frames.db
-pulsemaps:
-  - SRTInIcePulses
-features:
-  - dom_x
-  - dom_y
-  - dom_z
-  - dom_time
-  - charge
-  - rde
-  - pmt_area
-truth:
-  - energy
-  - position_x
-  - position_y
-  - position_z
-  - azimuth
-  - zenith
-  - pid
-  - elasticity
-  - sim_type
-  - interaction_type
+features: [sensor_pos_x, sensor_pos_y, sensor_pos_z, t]
+graph_definition:
+  arguments:
+    columns: [0, 1, 2]
```

Comment:
I think the naming of this "columns" arguments (which I understand as the columns included in the "distance" calculation of your edge definition) becomes very vague. At first glance it hard to tell what this feature does. Something like "edge_defining_columns" would be more descriptive I think.

Reply:
I agree that the name of that argument is not unambiguous, but I also fail to find better alternatives that are not very long.  The name of this argument is the same as what we used to call it in the KNNGraphBuilder (see h[ere](https://github.com/graphnet-team/graphnet/blob/becb3e62425aa77676797dc6dfb91354d68568b1/src/graphnet/models/graph_builders.py#L25)) and I do think that the doc string for this argument (see [here](https://github.com/graphnet-team/graphnet/blob/af943658f7d6c5ec5f79617b41b751e3ebfaee6c/src/graphnet/models/graphs/graphs.py#L34)) is pretty clear. So even though it's a bit challenging to understand what that argument does when one reads the config file, I think we should keep it as-is and refer users to the docs instead (which is the intended usage anyway). Is that OK with you? @Aske-Rosted

Reply:
Completely fine.